app/screens/settings/notification_mention/notification_mention.tsx

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useReducer, useState} from 'react';
+import {Alert, ScrollView, Text, View} from 'react-native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import FormattedText from '@components/formatted_text';
+import Section from '@components/section';
+import SectionItem from '@components/section_item';
+import {useTheme} from '@context/theme';
+import {t} from '@i18n';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import type UserModel from '@typings/database/models/servers/user';
+
+const UPDATE_MENTION_PREF = 'UPDATE_MENTION_PREF';
+const INITIAL_STATE = {
+    firstName: false,
+    usernameMention: false,
+    channel: false,
+};
+type Action = {
+    type: string;
+    data: Partial<typeof INITIAL_STATE>;
+}
+const reducer = (state: typeof INITIAL_STATE, action: Action) => {
+    switch (action.type) {
+        case UPDATE_MENTION_PREF:
+            return {
+                ...state,
+                ...action.data,
+            };
+
+        default:
+            return state;
+    }
+};
+
+const mentionHeaderText = {
+    id: t('notification_settings.mentions.wordsTrigger'),
+    defaultMessage: 'WORDS THAT TRIGGER MENTIONS',
```

Comment:
I know this was all caps in v1, but I'm totally against this, it should be Sentence case, the same for the other ones. if all caps is needed in the end, we can apply the text transform instead.

Reply:
done